docusaurus.config.js

```diff
@@ -204,65 +204,65 @@ const config = {
-                  <img class="slack" 
-                  src="https://raw.githubusercontent.com/KPRoche/iconography/refs/heads/main/assets/slack-mark-white.png" 
+                  <img class="slack"
+                  src="https://raw.githubusercontent.com/KPRoche/iconography/refs/heads/main/assets/slack-mark-white.png"
                   alt="Slack Icon" width="30px" height="auto" margin="5px"/>
                 </a>
               `,
               },
               {
                 html: `
                 <a href="https://inviter.co/llm-d-slack" target="_blank" rel="noreferrer noopener" aria-label="Inviter Icon">
-                  <span class="button-link">Join our Slack</span>    
+                  <span class="button-link">Join our Slack</span>
                   <img class="inviter"
-                    src="https://raw.githubusercontent.com/KPRoche/iconography/refs/heads/main/assets/inviter-logo.png" 
+                    src="https://raw.githubusercontent.com/KPRoche/iconography/refs/heads/main/assets/inviter-logo.png"
                     alt="Slack Inviter link" width="30px" height="auto" margin="5px"/>
                 </a>
               `,
               },
-              
+
               {
                 html: `
                 <a href="https://www.reddit.com/r/llm_d/" target="_blank" rel="noreferrer noopener" aria-label="Reddit Icon">
-                  <img class="reddit" 
-                    src="https://raw.githubusercontent.com/KPRoche/iconography/refs/heads/main/assets/reddit-mark-white.png" 
-                    alt="Reddit Icon" 
-                    width="30px" 
-                    height="auto" 
+                  <img class="reddit"
+                    src="https://raw.githubusercontent.com/KPRoche/iconography/refs/heads/main/assets/reddit-mark-white.png"
+                    alt="Reddit Icon"
+                    width="30px"
+                    height="auto"
                     margin="5px"/>
                 </a>
               `,
               },
               {
                 html: `
                 <a href="https://x.com/_llm_d_" target="_blank" rel="noreferrer noopener" aria-label="X Icon">
-                  <img class="x" 
-                  src="https://raw.githubusercontent.com/KPRoche/iconography/refs/heads/main/assets/x-mark-white.png" 
-                  alt="X Icon" 
-                  width="30px" 
+                  <img class="x"
+                  src="https://raw.githubusercontent.com/KPRoche/iconography/refs/heads/main/assets/x-mark-white.png"
+                  alt="X Icon"
+                  width="30px"
                   height="auto" margin="5px"/>
                 </a>
               `,
               },
             ],
           },
         ],
-        // copyright: `Copyright © ${new Date().getFullYear()} My Project, Inc. Built with Docusaurus.`,
+        copyright: `Copyright © ${new Date().getFullYear()} llm-d project. Sponsored by Red Hat`,
```

Comment:
This is the wording we need to confirm. (I took this from the instructlab project)